apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-steps/plan-step/index.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import LaunchStep, { Props as LaunchStepProps } from '../../launch-step';
+import './styles.scss';
+
+const PlanStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep } ) => {
+	const handleBack = () => {
+		onPrevStep && onPrevStep();
+	};
+
+	return (
+		<LaunchStep className="domain-step">
+			<div className="launch-step__header">
+				<div className="launch-step__heading">
+					<h1 className="launch-step__title">{ __( 'Choose a plan', 'full-site-editing' ) }</h1>
+					<p className="launch-step__subtitle">
+						{ __(
+							'Pick a plan that’s right for you. Switch plans as your needs change. There’s no risk, you can cancel for a full refund within 30 days.',
+							'full-site-editing'
+						) }
+					</p>
+				</div>
+				<div className="launch-step__actions">
+					<Button isTertiary onClick={ handleBack }>
+						{ __( 'Go back', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **136** times:
`translate( 'Go back' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/fse-editor-site-launch)